### PR TITLE
added travis continuous integration build file

### PR DESCRIPTION
**Pull request by timeraider4u**
Travis-CI can be used for free by open-source projects hosted on GitHub. Log-in at https://travis-ci.org/profile/ with your GitHub account. Click "Sync account" and then activate for mkvtoolnix repository to enable automatic Travis continuous integration builds for pushes and pull requests. At the moment Qt5 GUI feature compilation is disable until I find a ppa which contains >=Qt5.3.

### CODE REVIEW

**Comment by mbunkus** [comment]
Thanks, but I don't think I'll merge this. The most important reasons is that I'm already running a [buildbot instance](https://mkvtoolnix.download/buildbot/grid). That instance not only builds each commit, it does so on all the Linux versions I'm providing binaries for. It even builds the whole packages, going quite a bit farther than what the proposed Travis CI script would do. Your Travis CI file even tries to build on a Linux version that I simply don't support anymore.

Last but not least I've never used Travis CI myself. It'd be one more system I'd had to learn, to test, and to maintain support for.

Basically using Travis CI wouldn't offer anything while requiring time.

**Comment by mbunkus** [comment]
Done.
